.github/workflows/mattermost.yml

```diff
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
```suggestion
# This is a basic workflow to help you get started with Actions
```
This comment is not valid.

Please add a description what this action does and if it has inputs, what the inputs are etc.